content/monitors/notifications.md

```diff
@@ -27,98 +27,139 @@ right people get notified so the problem can be resolved as soon as possible.
+1. Displaying a different message depending on the
+notification type (e.g. triggered, recovered, no data)
+2. Incorporating the triggering scope into the message of multi alerts.
 
-{{< img src="monitors/notifications/conditionalvars.png" alt="conditional vars" responsive="true" popup="true" >}}
+#### Tag variables
 
-Here is an example of how you can set it up in the editor:
+When your monitor is a multi alert, instead of having a generic message (and finding the triggering tag scope in the alert query definition), a variable can be used in the message for explicitly identifying the triggering scope.
 
-{{< img src="monitors/notifications/templateconditionaleditor.png" alt="template conditional editor" responsive="true" popup="true" >}}
+Here is an example of how you can use template variables for a multi alert:
 
-The corresponding trigger event notification looks like this:
+{{< img src="monitors/notifications/templatevareditor.png" alt="template var editor" responsive="true" popup="true" >}}
 
-{{< img src="monitors/notifications/templateconditionaltrigger.png" alt="template conditional trigger" responsive="true" popup="true">}}
+and the corresponding event notification:
 
-and the recovery notification:
+{{< img src="monitors/notifications/templatevar.png" alt="template var" responsive="true" popup="true" >}}
 
-{{< img src="monitors/notifications/templateconditionalrecover.png" alt="template conditional recover" responsive="true" popup="true" >}}
+**Note**:  
+
+* The tag template variables available depend on the tag group selected in Step 1 of the monitor editor. The possible options automatically populate at the bottom of the "Use message template variables" help box in Step 3 of the editor.  
+
+* Template variable content is escaped by default. If your variable
+contains JSON or code that you would NOT like to be escaped, use triple braces instead of double braces (e.g. `{{{event.text}}}`).
+
+* See a complete list of contextual template variables available to your monitor by clicking the "Use message template variables" link or in the list of suggestions that appears when you type `{{` to begin a template variable name.
+
+* The tag template variables can also be used in the monitor titles (names), but the variables are only populated in the text of Datadog child events (not the parent, which displays an aggregation summary). 
+
+* Some tags identifying your triggering scope are automatically inserted into the title of your multi alert. If your scope is defined by a lot of tags, your alert title may end up being undesirably long.
+
+#### {{comparator}} template variables
+
+The `{{comparator}}` template variable's value is always a relational operator. It correlates to the relational value selected in the monitor's "Set alert conditions" section:
```

Comment:
Should it be:

```
It corresponds to the relational value selected in the monitor's "Set alert conditions" section:
```

instead of

```
It correlates to the relational value selected in the monitor's "Set alert conditions" section:
```

Reply:
👍